testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -36,9 +36,7 @@
     assert_array_almost_equal,
     assert_raises,
```

Comment:
can you remove this import as well. Seems like you already changed all occurrences of this